lyricsgenius/genius.py

```diff
@@ -322,7 +322,10 @@ def search_album(self, name=None, artist="",
                 print("No results found for: '{s}'".format(s=search_term))
             return None
 
-        album_id = album_info['id']
+        if album_id is None and get_full_info is True:
+            album_id = album_info['id']
```

Comment:
This line should be out of the if statement since the `album_id` is needed afterwards and it would be None in the case where `album_id=None` and `get_ful_info=False`.

Reply:
Oh, that's right! Sorry! :sweat_smile: